docker/Dockerfile.demo_vitis_ai

```diff
@@ -16,7 +16,7 @@
 # under the License.
 
 # CI docker VAI env
-FROM xilinx/vitis-ai:latest
+FROM xilinx/vitis-ai:1.4.916
```

Comment:
This is the main Vitis AI docker used in the documentation and this docker isn't used for CI as we are only running tests within the `ci_cpu` docker. So, the change doesn't need to be in the other PR. I noticed the line above says `# CI docker VAI env`, which is still there from the original `ci_vitis_ai` name we used. I changed it to avoid confusion.

Reply:
Ah yeah. You're right.